gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/provider/JsonParseExceptionMapper.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.management.handlers.management.api.provider;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import io.gravitee.am.management.handlers.management.api.model.ErrorEntity;
+import io.gravitee.common.http.HttpStatusCode;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import jakarta.ws.rs.ext.Provider;
+
+@Provider
+public class JsonParseExceptionMapper extends AbstractExceptionMapper<JsonParseException> {
+    @Override
+    public Response toResponse(JsonParseException e) {
+        return Response
```

Comment:
suggestion: it maybe useful to get the exception stacktrace/message in a log (at least DEBUG) wdyt ?